### PR TITLE
Enforce types 

### PR DESCRIPTION
**Pull request by b1zzu**
**Reason**:
this PR aims to remove all `@ts-ignore`  and enforce types. 

The main change is the introduction of the `Device` object ( util/device.ts ) which expose the `execute()` method which is a wrapper around the `executeAsync()` method

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,25 +10,32 @@
...
-    "graphql": "latest",
-    "mocha": "^6.1.4",
-    "node-fetch": "^2.6.0",
-    "pg": "^7.11.0",
+    "@types/node-fetch": "2.5.0",
+    "@types/pg": "7.4.14",
+    "axios": "0.19.0",
+    "chai": "4.2.0",
+    "express": "4.17.1",
+    "graphql": "14.4.2",
```
Comment (jhellar) [inline package.json]: Should not we use latest for `express` and `graphql`? That is what user will eventually install.

File: scripts/setup-env-vars.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ else
   export UPS_HOST=bs-local.com
   export UPS_PORT=8089
   
-  export PGHOST=localhost
+  export PGHOST=bs-local.com
```
Comment (jhellar) [inline scripts/setup-env-vars.sh]: Why was this changed to `bs-local.com`? Postgres is accessed from test script, not from device running in browser stack.

File: package.json (hunk 1 above)
Comment (b1zzu) [reply package.json]: Yes, ist's better to test with the latest version of graphql and express, but at the same time if we always use the latest it will be difficult to understand if a new issue will be caused by a new version of express/graphql or to new changes to the aerogear packages, in any case, this problem could need a wider analysis so for now I will revert it to `latest`

File: scripts/setup-env-vars.sh (hunk 2 above)
Comment (b1zzu) [reply scripts/setup-env-vars.sh]: at the end `bs-local.com` resolves to `127.0.0.1` which is the same as for `localhost`. For example for keycloak where some operations are performed from the test script and some from the device, they both use the host `bs-local.com`. I mainly wanted to unified everything to one single host but it doesn't make much different
